sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsErrorTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Testing;
+using NUnit.Framework;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace Azure.AI.TextAnalytics.Tests
+{
+    public class TextAnalyticsErrorTests : SyncAsyncPolicyTestBase
+    {
+        public TextAnalyticsErrorTests(bool isAsync) : base(isAsync) { }
+
+        [Test]
+        public void DeserializeTextAnalyticsError()
+        {
+            var errors = @"
```

Comment:
Could be worth adding a test where no innerError is returned. I think sending bad credentials will do this.